src/Renderer/AirspaceRendererOther.cpp

```diff
@@ -67,9 +67,10 @@ class AirspaceVisitorMap final
     if (warnings.IsAcked(airspace))
       return;
 
-    AirspaceClass airspace_class = airspace.GetClass();
+    AirspaceClass asclass = airspace.GetType() ? airspace.GetClass() : airspace.GetType();
+
     if (settings.fill_mode == AirspaceRendererSettings::FillMode::NONE ||
-        settings.classes[airspace_class].fill_mode ==
+        settings.classes[asclass].fill_mode ==
```

Comment:
Yet much more noise just for renaming a local variable. Please don't do that. If you feel a variable should be renamed, don't mix it with a feature change.